psscriptanalyzer/Common-Helpers.psm1

```
@@ -0,0 +1,58 @@
+    foreach ($url in $Urls) {
+        $r = iwr -Uri $url -UseBasicParsing
+        if ($r -eq $null) {
+            Write-Host "$url unreachable"
+        }
+    }
+}
+
+function Process-Tokens {
+    param(
+        [string]$Username,
+        [string]$Password,
```

PSScriptAnalyzer (1.25.0)

[error] Function 'Process-Tokens' has both Username and Password parameters. Either set the type of the Password parameter to SecureString or replace the Username and Password parameters with a Credential parameter of type PSCredential. If using a Credential parameter in PowerShell 4.0 or earlier, please define a credential transformation attribute after the PSCredential type attribute.

(PSAvoidUsingUsernameAndPasswordParams)